packages/@aws-cdk/aws-cloudformation/lib/custom-resource.ts

```diff
@@ -160,11 +161,17 @@ export class CustomResource extends Resource {
 
     const type = renderResourceType(props.resourceType);
     const providerConfig = props.provider.bind(this);
+    
+    // change properties to upper case under a feature flag (since this is a breaking change).
+    const properties = this.node.tryGetContext(cxapi.DISABLE_CUSTOM_RESOURCE_UPPERCASE_PROPERTIES)
```

Comment:
Can't use a feature flag for this because it can break "legacy" constructs used in "new" applications.